package.json

```diff
@@ -38,7 +38,6 @@
   },
   "dependencies": {
     "chalk": "^2.1.0",
-    "generator-license": "^5.4.0",
```

Comment:
what's the reasoning for removing this and implementing our own? Just curious

Reply:
There's no way to get the value a user inputs for the license prompt from a sub-generator. And we need it to template `manifest.json`. The `generator-license` sub-generator only handles putting it into the `package.json`. Sadly we can't tell it to put it in `manifest.json` too.